hr_employee_calendar_planning/hooks.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2019 Tecnativa - Pedro M. Baeza
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, SUPERUSER_ID
+from collections import defaultdict
+
+
+def post_init_hook(cr, registry, employees=None):
```

Comment:
@pedrobaeza during the migration to v11 we realized that the previous init hook was not working properly, and was producing overlapping calendars, and so it would crash. We have simplified, as it can be really complex to migrate without overlaps. I think that each company should consider installing this module and then deciding how to best convert their existing calendars.

Reply:
Not really. On "fragmented" calendars, there's no overlap. The overlap on employees might happen if your original calendar had it, but simply deactivate the constraint through a context key on that init hook, but please, don't remove this as good initial setup.

Reply:
Now the old logic has been maintained.